<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>433</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>532</width>
    <height>433</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>532</width>
    <height>433</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Sky Box Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="1" column="0">
      <widget class="QGraphicsView" name="graphicsView_back"/>
     </item>
     <item row="1" column="1">
      <widget class="QGraphicsView" name="graphicsView_right"/>
     </item>
     <item row="1" column="2">
      <widget class="QGraphicsView" name="graphicsView_front"/>
     </item>
     <item row="1" column="3">
      <widget class="QGraphicsView" name="graphicsView_left"/>
     </item>
     <item row="0" column="1">
      <widget class="QGraphicsView" name="graphicsView_top"/>
     </item>
     <item row="2" column="1">
      <widget class="QGraphicsView" name="graphicsView_bottom"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
